### PR TITLE
fix: default nice to false if domainMin/domainMax are set

### PR DESCRIPTION
**Pull request by maclockard**
fixes https://github.com/vega/vega-lite/issues/7796

### CODE REVIEW

File: site/_data/versions.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 vega: 5.21.0
 vega-lite: 5.1.1
-vega-embed: 6.19.1
+vega-embed: 6.20.0
```
Comment (maclockard) [inline site/_data/versions.yml:3]: I have no idea why this bumped when I ran `yarn install && yarn test`

File: src/scale.ts [hunk 2]
```diff
@@ -673,7 +673,7 @@ export interface Scale<ES extends ExprRef | SignalRef = ExprRef | SignalRef> {
    *
    * For temporal fields with time and utc scales, the `nice` value can be a string indicating the desired time interval. Legal values are `"millisecond"`, `"second"`, `"minute"`, `"hour"`, `"day"`, `"week"`, `"month"`, and `"year"`. Alternatively, `time` and `utc` scales can accept an object-valued interval specifier of the form `{"interval": "month", "step": 3}`, which includes a desired number of interval steps. Here, the domain would snap to quarter (Jan, Apr, Jul, Oct) boundaries.
    *
-   * __Default value:__ `true` for unbinned _quantitative_ fields; `false` otherwise.
+   * __Default value:__ `true` for unbinned _quantitative_ fields without explicit domain bounds; `false` otherwise.
```
Comment (maclockard) [inline src/scale.ts:676]: Was unsure of how to best phrase this

**Comment by maclockard** [comment]
@domoritz I added `GH_PAT` secret personal access token to my fork with the following permissions: 
Screen Shot 2021-11-12 at 10 59 08 AM
but it looks like actions still arn't running? is there a step I'm missing?

**Comment by domoritz** [comment]
I'm not sure. Maybe the instructions aren't complete. If you could investigate, that would be great.

File: site/_data/versions.yml (hunk 1 above)
Comment (domoritz) [reply site/_data/versions.yml:3]: This happens when you run `yarn build:versions`. We usually run it before releases.

**Comment by domoritz** [comment]
I had to approve running the tests since this is your first contribution to Vega-Lite.

File: site/_data/versions.yml (hunk 1 above)
Comment (maclockard) [reply site/_data/versions.yml:3]: ah, okay, I'll revert this file then.

File: src/scale.ts (hunk 2 above)
Comment (domoritz) [reply src/scale.ts:676]: Sounds good.

**Comment by domoritz** [comment]
Moved to https://github.com/vega/vega-lite/pull/7802.

**Pull request closed without merging**